src/databricks/labs/ucx/source_code/linters/pyspark.py

```diff
@@ -328,24 +327,24 @@ def name(self) -> str:
         return self._from_table.name()
 
     def lint(self, code: str) -> Iterable[Advice]:
-        linter = ASTLinter.parse(code)
-        for node in TreeWalker.walk(linter.root):
+        tree = Tree.parse(code)
+        for node in TreeWalker.walk(tree.root):
```

Comment:
```suggestion
        for node in tree.walk():
```
Technically, walk can be __iter__